sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERAPPDELEGATE_H_
```

Comment:
Since we generally `#include` files, I add these guards explicitly. The headers we vendor may or may not be `import`ed by clients as well. Doesn't hurt to keep these around.